composer.json

```diff
@@ -37,7 +37,7 @@
         "source": "https://github.com/MyIntervals/emogrifier"
     },
     "require": {
-        "php": "~7.0 || ~7.1 || ~7.2 || ~7.3 || ~7.4",
+        "php": "~7.1 || ~7.2 || ~7.3 || ~7.4",
```

Comment:
Noting that [the `.0` point version suffix](https://getcomposer.org/doc/articles/versions.md#tilde-version-range-) should be restored at some point so the behaviour is actually as intended (though there will not be a PHP 7.5).

Reply:
Agreed.

Reply:
#881